src/JoomlaBrowser.php

```diff
@@ -49,6 +84,8 @@ class JoomlaBrowser extends WebDriver
 	public function doAdministratorLogin($user = null, $password = null)
 	{
 		$I = $this;
```

Comment:
Can we get rid of the $I = $this?

the benefit of doing this is really small.. If you want in a second PR :)